modules/client/examples/02-multisig-client/07-execute-proposal.ts

```diff
@@ -1,6 +1,6 @@
 /* MARKDOWN
 ---
-title: Execution
+title: Execute Proposal Action
```

Comment:
You don't execute a specific action. You execute all the actions from the proposal
```suggestion
title: Execute Proposal
```